library/src/main/java/org/antennapod/audio/SonicAudioPlayer.java

```diff
@@ -777,7 +813,10 @@ public void run() {
                 }
                 Log.d(TAG_TRACK, "Decoding loop exited. Stopping codec and track");
                 Log.d(TAG_TRACK, "Duration: " + (int) (mDuration / 1000));
-                Log.d(TAG_TRACK, "Current position: " + (int) (mExtractor.getSampleTime() / 1000));
+
+                if (!(mIsInitiating || mIsSeeking)) {
+                    Log.d(TAG_TRACK, "Current position: " + (int) (mExtractor.getSampleTime() / 1000));
```

Comment:
Wouldn't it make more sense to call ``getCurrentPosition()`` here? We would probably have to handle more states there (at least ``STATE_PREPARING``), but ``mIsInitiating`` seems unnecessary.